wp-parsely.php

```diff
@@ -21,6 +21,67 @@
 	'3.1',
 ];
 
+/**
+ * Keep track of Parse.ly loading info in one place.
+ */
+final class Parsely_Loader_Info {
+	// Describes how the parse.ly plugin is integrated/loaded.
+	const INTEGRATION_TYPE_MUPLUGINS        = 'MUPLUGINS';
+	const INTEGRATION_TYPE_MUPLUGINS_SILENT = 'MUPLUGINS_SILENT';
+	const INTEGRATION_TYPE_SELF_MANAGED     = 'SELF_MANAGED';
+	const INTEGRATION_TYPE_NONE             = 'NONE';
+
+	// Defaults for when detection was not possible.
+	const VERSION_UNKNOWN      = 'UNKNOWN';
+	const SERVICE_TYPE_UNKNOWN = 'UNKNOWN';
+
+	private static bool $active;
+	private static string $integration_type;
+	private static string $service_type;
+	private static string $version;
+	private static array $parsely_options;
+
+	public static function is_active(): bool {
+		return isset( self::$active ) ? self::$active : false;
+	}
+
+	public static function set_active( bool $active ): void {
+		self::$active = $active;
+	}
+
+	public static function get_integration_type(): string {
+		return isset( self::$integration_type ) ? self::$integration_type : self::INTEGRATION_TYPE_NONE;
+	}
+
+	public static function set_integration_type( string $integration_type ): void {
+		self::$integration_type = $integration_type;
+	}
+
+	public static function get_service_type(): string {
+		return isset( self::$service_type ) ? self::$service_type : self::SERVICE_TYPE_UNKNOWN;
+	}
+
+	public static function set_service_type( string $service_type ): void {
+		self::$service_type = $service_type;
+	}
```

Comment:
Should we strip these out until/if we need them? Not a big fan of shipping unused "maybe we use this in the future" type code. Always easier to add than to remove later :)

Reply:
IMO we can strip these out.

Reply:
I have removed this in the latest commit.

Reply:
I will mirror that change in the SDS code.